### ISSUE TITLE
Use rootDir instead of srcDir as target directory for generated files

### ISSUE THREAD

**Issue by Pierre-M** [victorgarciaesgi/nuxt-typed-router#59]
Let's say my project architecture is the following with srcDir differing from rootDir in Nuxt config: 

```
.nuxt
nuxt.config.ts
/webapp
  ... all my front codebase goes here
```

And in my Nuxt config : 

```typescript
export default defineNuxtConfig({
  rootDir: "." // by default
  srcDir: "./webapp"
})
```

The issue is : 

* typescript alias for @typed-router points to .nuxt/typed-router
* generated files are located in /webapp/.nuxt/typed-router

I'm not sure why it's not the case, but shouldn't you use `rootDir` Nuxt options instead of `srcDir` Nuxt option to handle generated files location ?

But thanks for your work, your plugin is really great 🤗

**Comment by victorgarciaesgi** [comment]
Hi! 
Thanks for your issue!

Oh yeah that's my bad, I will make a quick fix and keep you updated :D 

By the way i'm already planning on releasing soon a `v2.1.0` where it's no longer needed to use `useTypedRouter` or `useTypedRoute` because I managed to override the typings of `useRoute` and `useRouter`

**Comment by Pierre-M** [comment]
Ooooh nice one ! I'll wait for the 2.1.0 then 🤗

**Comment by victorgarciaesgi** [comment]
`v2.0.3` just released! Tell me if it fixed your problem, i tested it on my playground.
Thnaks for the issue because I didn't though of it

**Comment by Pierre-M** [comment]
Yeaye ! It works fine thks a lot !

**Comment by victorgarciaesgi** [comment]
If you want to test the next version, you can already with `v2.1.0-beta.4`. I still need to write rebust e2e and unit tests

**Comment by tobychidi** [comment]
The latest stable release breaks my app. 

 Error while requiring module nuxt-typed-router: TypeError [ERR_INVALID_FILE_URL_PATH]: File URL path must be absolute

**Comment by victorgarciaesgi** [comment]
Do you reproduce it on the stackblitz playground?

**Comment by tobychidi** [comment]
I use node v18.11.0

**Comment by victorgarciaesgi** [comment]
It's don't think it's linked to node version, can't reproduce with node 18 in local.
Maybe clear your node_modules and `.nuxt` folder and retry

**Comment by tobychidi** [comment]
Just tired with the stackblitz. Same error https://stackblitz.com/edit/nuxt-starter-pdxokn?file=nuxt.config.ts

**Comment by tobychidi** [comment]
> It's don't think it's linked to node version, can't reproduce with node 18 in local. Maybe clear your node_modules and `.nuxt` folder and retry

Did this already

**Comment by victorgarciaesgi** [comment]
Fixed and pushed in `v2.1.1` thanks for your report @tobychidi !

**Comment by tobychidi** [comment]
You're welcome
